src/it/release-finish-8-it/invoker.properties

```diff
@@ -0,0 +1,3 @@
+invoker.goals=${project.groupId}:${project.artifactId}:${project.version}:release-finish -DpushRemote=false -DuseSnapshotInRelease=true -DcommitDevelopmentVersionAtStart=true -DskipMergeDevBranch=true
```

Comment:
Parameters `useSnapshotInRelease` and `commitDevelopmentVersionAtStart` are not related to this test. Please adjust tests and description.